web-marketplace/src/pages/Projects/AllProjects/AllProjects.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react';
 import { useLocation, useNavigate, useParams } from 'react-router-dom';
```

Comment:
we will need some config to filter only terrasos projects (BT01 class) for the terrasos app but that's probably too early to implement, maybe there will be more credit classes or off-chain projects

Reply:
Ok, we'll be able to handle that inside with `Clients.config.ts`, I think.